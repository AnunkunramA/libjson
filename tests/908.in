[ "6EiK4DAzVLvMTjr", "n7yMpW\"sRgviyqdp", { "dyYNjmJWtQV" : 23507700.4073602, "sEWOlpMNjMKEoBXvl" : { "7PbcKvxwZYm3DmWwwiJ" : null, "
9FO" : [ 62863849.9167323, null, 34236045.5824622, "OVRDjCPbB", null, 48193352.6158155,  ], "BrYkEvDP0" : "4QInt6yBckO8TPVdA", "\"wgoCheTz" : null, "NK\"U
Pib5	oZMA" : false, "7hn\"\"KQfC
QQw8Nqa" : true,  }, "nuBjb4PY3Ty\"V" : null,  }, null, [ true, "TigQa", "3S", [ null, false,  ], null, 71565914.3302464, [ { "wiNo" : 79796304.7607531, "FNJdI
i3PqubV0i" : { "wc" : false, "lqHMrwcwhm7H8n" : "Cce	eM",  }, "stqsF7bVtqT " : 17482074.0333647, "2b4Q" : false, "00qksvsZo" : { "g5S6hl9PYP9" : true,  }, "5\"xhbHi" : null, "C3f11uG5Pyx
3" : true, "BI	cL\"PN1gu
rK" : null,  },  ], null,  ],  ]
