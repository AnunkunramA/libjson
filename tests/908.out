["6\fEiK4DAzVLvMTjr", "n7yMpW\"sRg\fviyqdp", {"dyYNjmJWtQV":2.35077e+07, "sEWOlpMNjMKEoBXvl":{"7PbcKvxwZYm3DmWwwiJ":null, "
9FO":[6.28638e+07, null, 3.4236e+07, "OVRDjCPbB", null, 4.81934e+07], "BrYkEvDP0":"4QInt6yBckO\r8TPVdA", ""wgoCheTz":null, "NK"U
Pib5	oZMA":false, "7hn""KQfC
QQw8Nqa":true}, "nuBjb4PY3Ty"V":null}, null, [true, "Tig\fQa", "3S", [null, false], null, 7.15659e+07, [{"wiNo":7.97963e+07, "FNJdI
i3PqubV0i":{"wc":false, "lqHMrwcwhm7H8n":"Cce\teM"}, "stqsF7bVtqT ":1.74821e+07, "2b4Q":false, "00qksvsZo":{"g5S6hl9PYP9":true}, "5"xhbHi":null, "C3f11uG5Pyx
3":true, "BI	cL"PN1gu
rK":null}], null]]
